xonsh/built_ins.py

```diff
@@ -23,7 +23,7 @@
 
 from xonsh.ast import AST
 from xonsh.lazyasd import LazyObject, lazyobject
-from xonsh.history import History
+from xonsh.history import get_history_backend
```

Comment:
This should be something like, 

```python
import xonsh.history as xh
```

like we do with other imports that are outside of the current subpackage. See https://github.com/xonsh/amalgamate for more details